spark/src/main/scala/org/allenai/pipeline/spark/PartitionedRddIo.scala

```diff
@@ -0,0 +1,102 @@
+package org.allenai.pipeline.spark
+
+import org.allenai.common.Logging
+import org.allenai.pipeline._
+import org.apache.hadoop.fs.{ FileSystem, Path }
+import org.apache.hadoop.mapred.JobConf
+import org.apache.spark.SparkContext
+import org.apache.spark.SparkContext._
+import org.apache.spark.rdd.RDD
+
+import scala.reflect.ClassTag
+
+/** Created by rodneykinney on 5/24/15.
+  */
+class PartitionedRddIo[T: ClassTag: StringSerializable](
```

Comment:
Similarly, we now have a single IO class for doing the persistence, instead of having separate implementations for files and s3 blobs.